### PR TITLE
.coafile: Remove pycodestyle_ignore list items

### PR DESCRIPTION
**Pull request by jgarciabu**
This change removes specified items from the pycodestyle_ignore variable

Closes https://github.com/coala/coala/issues/5389

### CODE REVIEW

**Comment by z1ggy-o** [comment]
Maybe you need to change the commit's *shortlog*.  
By documentation about [commit](https://api.coala.io/en/latest/Developers/Writing_Good_Commits.html), the shortlog should starts with a capital letter, and be written in imperative present tense.

File: .gitignore [hunk 1]
```diff
@@ -52,3 +52,4 @@ GTAGS
 GRTAGS
 GPATH
 GSYMS
+/.vs
```
Comment (z1ggy-o) [inline .gitignore]: Why add this into .gitignore?
It does not seem to be related to the issue.

**Comment by jgarciabu** [comment]
I'm unsure how to keep Visual Studio from creating that file. When I choose to ignore the vs folder that's made when I make updates, it creates that entry in the file.

File: .gitignore (hunk 1 above)
Comment (jayvdb) [reply .gitignore]: I agree with @z1ggy-o, this change needs to be removed from your commit.

It is a separate problem. and should not be included in your PR.

Comment (jgarciabu) [reply .gitignore]: I removed the addition in line right from Github and it looks like it passed all tests.

I will figure out how to prevent that from happening at the source. Let me know if I need to provide additional edits.

**Comment by z1ggy-o** [comment]
You may need to squash the commits and modify you commit message to follow the commit rules.
[Here is the tutorial](http://api.coala.io/en/latest/Developers/Git_Basics.html) about how to do that.

**Comment by jgarciabu** [comment]
@z1ggy-o Please review my commit message at the top of the pull request. It adheres to the instructions found here: https://api.coala.io/en/latest/Developers/Writing_Good_Commits.html

It may have looked odd before because I didn't remove the auto-inserted text when one opens a pull request for this project. I didn't know if that would have been suppressed when I submitted the request, so I left it. It didn't suppress the boilerplate text and so that's why my commit message looked incorrect. I've since removed the boilerplate and the remaining message adheres to the standards, unless there are different standards now being used. Please advise specifically what I would need to do from now to get the PR merged.

**Comment by jgarciabu** [comment]
Ok, how does that look now?

**Comment by jayvdb** [comment]
Your commit message is not correct.  You need to use imperative tense for the shortlog.

**Comment by jgarciabu** [comment]
Please forgive my ignorance. I'm still new to contributing to git/visual studio/contributing to open source projects. I have read the tutorial and it's not difficult to follow what's required. What's confusing is how to submit changes to commit messages from Visual Studio when you don't necessarily have to make changes to any file in the code repository. Please be patient with me as I attempt to put through this change.

**Comment by palash25** [comment]
> but the barrier to entry on the changes is very high for a new comer

That is just our way to train developers.

Please don't give up @jgarciabu I can  help you out with this PR if you are willing to work again. It isn't that tough to follow the commit guidelines . I had the same problems with my first PR but didn't give up the issue. I have been contributing for about a year now.

So please give it another shot.

**Comment by jgarciabu** [comment]
If I can delete this pull request and start over, I will try again.

**Comment by palash25** [comment]
Now go read this guide on how to squash your commits http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html

Once you enter the command in your terminal you will be taken to a screen listing all the commits that you want to squash into. Just delete all the unnecessary commit messages and keep the one with the appropriate message.

Once you are done squashing rebase

`git fetch origin`
`git rebase origin/master`

then force push

`git push -f myfork <branch-name>`

**Comment by jgarciabu** [comment]
I've broken something on my end. I squashed into someone else entirely's commits. I don't know how that's possible but it is. I apparently squashed everything into e775013e on my end.

**Comment by palash25** [comment]
Hey can you ask on the coala/coala channel on gitter its hard to help you out on this thread. We can all figure out whats the problem there.

Just explain it what you did and someone will get back to you.
